interchain/scripts/README.md

```diff
@@ -0,0 +1,55 @@
+# Abstract Scripts
+
+This crate contains scripts to deploy and interact with Abstract. This README contains multiple tutorials relate to Abstract deployments
+
+## Deploying Abstract IBC
+
+This is a unifying guide on how to deploy Abstract IBC between 2 chains.
+
+### Deploy Abstract
+
+The first step in deploying abstract is deploying Abstract on the 2 chains you want to connect.
+
+### Deploy and Connect Polytone
+
+Then you need to deploy and connect Polytone between the 2 chains. You should find a complete tutorial inside the [abstract-polytone repo](https://github.com/AbstractSDK/polytone/blob/abstract/deployment/DEPLOY.md).
+
+This comes in 3 steps:
+
+#### Deploy the Polytone contracts on each chain
+
+Upload the polytone contracts (voice, note and proxy at least) on each chain. A script is available in the [abstract-polytone repo](https://github.com/AbstractSDK/polytone/blob/abstract/deployment/scripts/src/bin/upload_contracts.rs).
+
+#### Instantiate the contracts on each chain
+
+In order to create a one-directional connection, you need to instantiate the note on the local side and the voice on the remote side. A script is available in the [abstract-polytone repo](https://github.com/AbstractSDK/polytone/blob/abstract/deployment/scripts/src/bin/instantiate_chains.rs).
```

Comment:
The deploy-ibc script deploys abstract-IBC on one chain. Now this is integrated in the deploy script, so this is not needed anymore
As per the second one, sure, why not :)

Reply:
Asking because it can reduce the number of steps significantly. In #333 it also ensures that when Abstract is deployed then the IBC infra is deployed. IN writing this, however, it doesn't guartantee that a loaded abstract instance has the IBC infra present, which could lead to inconsistencies.